### PR TITLE
Externalising the SSLSessionCacheSize property

### PR DESCRIPTION
**Pull request by seshadhri-aswath**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

################################################################################################

### CODE REVIEW

File: dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -35,3 +35,5 @@ default.sslRef.desc=The default SSL configuration repertoire. The default value
 ssl.sslSessionTimeout=SSL session timeout
 ssl.sslSessionTimeout.desc=The timeout limit for an SSL session that is established by the SSL Channel. 
 
+ssl.sslSessionCacheSize=SSL session Cache Size
```
Comment (helyarp) [inline dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties]: Suggested change: `SSL session cache size` (Use lowercase unless uppercase is needed.)

File: dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -35,3 +35,5 @@ default.sslRef.desc=The default SSL configuration repertoire. The default value
...
+ssl.sslSessionCacheSize.desc=Limit the number of allowed items in the cache.
```
Comment (helyarp) [inline dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties]: The "Limit" in the desc is a verb, when a noun is expected for a desc. Suggested change: 
`The maximum number of items allowed in the cache.`

If the desc should mean something different, pls tell me.

**Review by helyarp — COMMENTED** [review]
Suggested changes

File: dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (seshadhri-aswath) [reply dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties]: Updated to lowercase as suggested

File: dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (seshadhri-aswath) [reply dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties]: Made the change as suggested.

**Review by helyarp — DISMISSED** [review]
Looks good!

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#build

**Comment by seshadhri-aswath** [comment]
#libby #build

**Comment by seshadhri-aswath** [comment]
#libby #build

**Review by mrsaldana — APPROVED** [review]
LGTM

**Comment by seshadhri-aswath** [comment]
fixes [#28377](https://github.com/OpenLiberty/open-liberty/issues/28377)

**Pull request merged**